decisions/flowfixme.md

```diff
@@ -5,7 +5,7 @@ When we tried to upgrade to Flow 0.112, all of our components that used `...rest
 We decided to suppress these comments using the following comment:
 
 ```
-// $FlowFixMe - inexact rest. See 'decisions/flowfixme.md'.
+// $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
```

Comment:
As flow now expects an error code to be provided updating example comment for easy use.

Reply:
This is so much better, I'm happy they added this

Reply:
Yeah so cleaner